apps/server/src/modules/account/services/account.service.ts

```diff
@@ -75,6 +80,13 @@ export class AccountService {
 		return AccountEntityToDtoMapper.mapToDto(account);
 	}
 
+	async updateLastTriedFailedLogin(accountId: EntityId, lastTriedFailedLogin: Date): Promise<AccountDto> {
```

Comment:
feathers calls this method with `lastTriedFailedLogin: null` to reset the failed login. Thus it is and should be valid to give null as a parameter.